components/eamxx/src/physics/mam/eamxx_mam_dry_deposition_process_interface.cpp

```diff
@@ -223,67 +280,179 @@ void MAMDryDep::initialize_impl(const RunType run_type) {
+
+  dry_deposition.compute_tendencies(aero_config, team, t, dt, atm, surf, progs, diags, tends);
+}
+void compute_tendencies(
+    const int ncol, 
+    const int nlev,
+    const MAMDryDep::Parameters parameters,
+    const mam4::DryDeposition dry_deposition,
+    const double dt,
+    MAMDryDep::view_3d qtracers,
+    MAMDryDep::view_3d d_qtracers_dt,
+    const MAMDryDep::view_3d dgncur_awet_,
+    const MAMDryDep::view_3d wet_dens_,
+    const mam_coupling::DryAtmosphere dry_atm,
+    const mam_coupling::AerosolState dry_aero,
+    const mam_coupling::AerosolState wet_aero,
+    MAMDryDep::view_2d aerdepdrycw, 
+    MAMDryDep::view_2d aerdepdryis,
+    MAMDryDep::view_3d tendencies)
+{
+  const auto policy = ekat::ExeSpaceUtils<MAMDryDep::KT::ExeSpace>::get_default_team_policy(ncol, nlev);
+  Kokkos::parallel_for(policy, KOKKOS_LAMBDA(const MAMDryDep::KT::MemberType& team) {
+    const int num_aero_modes = mam_coupling::num_aero_modes();
+    MAMDryDep::view_2d dgncur_awet[num_aero_modes];
+    MAMDryDep::view_2d wet_dens[num_aero_modes];
+
+    for (int i=0; i<num_aero_modes; ++i)
+      dgncur_awet[i] = ekat::subview(dgncur_awet_, i);
+
+    for (int i=0; i<num_aero_modes; ++i)
+      wet_dens[i] = ekat::subview(wet_dens_, i);
+
+    compute_tendencies(team, parameters, dry_deposition, dt, qtracers, d_qtracers_dt, 
+      dgncur_awet, wet_dens, dry_atm, dry_aero, wet_aero, aerdepdrycw, aerdepdryis, tendencies);
+  });
+}
+
+void fill_tracer_views(
+    const int ncol, 
+    const int nlev, 
+    mam_coupling::DryAtmosphere dry_atm, 
+    mam_coupling::AerosolState dry_aero, 
+    mam_coupling::AerosolState wet_aero, 
+    MAMDryDep::view_3d qtracers) 
+{ 
+  const int num_aero_modes = mam_coupling::num_aero_modes();
+  const int num_aero_species = mam_coupling::num_aero_species();
+  const auto policy = ekat::ExeSpaceUtils<MAMDryDep::KT::ExeSpace>::get_default_team_policy(ncol, nlev);
+  Kokkos::parallel_for(policy, KOKKOS_LAMBDA(const MAMDryDep::KT::MemberType& team) {
+    const int column = team.league_rank(); 
+    compute_wet_mixing_ratios(team, dry_atm, dry_aero, wet_aero, column);
+
+    Kokkos::parallel_for(Kokkos::TeamVectorRange(team, nlev), [&] (const int lev) {
+      for (int mode = 0; mode < num_aero_modes; ++mode) {
+        int icnst = mam4::ConvProc::numptrcw_amode(mode);
+        qtracers(column, lev, icnst) = wet_aero.int_aero_nmr[mode](column,lev);
+        for (int species = 0; species < num_aero_species; ++species) { 
+          icnst = mam4::ConvProc::lmassptrcw_amode(species, mode);
+	  if (-1 < icnst) {
+            qtracers(column, lev, icnst) = wet_aero.int_aero_mmr[mode][species](column,lev);
+	  }
+	}
+      }
+    });
+  });
+}
+}
 
 // =========================================================================================
 void MAMDryDep::run_impl(const double dt) {
 
-const auto scan_policy = ekat::ExeSpaceUtils<
+  // Include this but update when scream is updated to use the newer DryDeposition class:
```

Comment:
Could you please explain this comment? I am not clear on what needs to be updated here. Thanks.